src/app/[locale]/api/keys/decrypt-secret/route.ts

```diff
@@ -0,0 +1,44 @@
+import type { NextRequest } from 'next/server';
+import { NextResponse } from 'next/server';
+
+import { getClientById } from '@/libs/actions/user';
+import { decrypt } from '@/service/crypto';
+
+export async function POST(
```

Comment:
make this GET request, clientid can be passed as request param

Reply:
Since this is will only be used to show the secret key in the UI, this should actually be a server action right?